### ISSUE TITLE
Better "how to run samples" instructions needed for Authentication samples

### ISSUE THREAD

**Issue by marknadig** [dotnet/aspnetcore#32734]
Instructions on how to run the samples are very sparse, and after trying a few things I never could. I had cloned the entire aspnetcore repo and tried to run and example from src\Security\samples:
`dotnet run --project DynamicSchemes`

that build failed due to missing nuget packages. had to
`dotnet restore ..\..\eng\tools\RepoTasks`

Now build just fails
`C:\Program Files\dotnet\sdk\5.0.202\Microsoft.Common.CurrentVersion.targets(4919,5): error MSB3030: Could not copy the file "C:\Development\aspnetcore\artifacts\bin\InProcessRequestHandler\Win32\Debug\aspnetcorev2_inprocess.dll" because it was not found. [C:\Development\aspnetcore\src\Servers\IIS\IIS\src\Microsoft.AspNetCore.Server.IIS.csproj]
`
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 82950389-524e-0754-a125-addeccbdb1b3
* Version Independent ID: 80b7f7cf-dc2e-137a-5aa7-fa88da63ac80
* Content: [Authentication samples for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/samples?view=aspnetcore-5.0)
* Content Source: [aspnetcore/security/authentication/samples.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/samples.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by blowdart** [comment]
I don't believe that issue is specific to our samples, rather engineering all up. @Tratcher whose responsibility would this be?

**Comment by Tratcher** [comment]
We have these two:
- https://github.com/dotnet/aspnetcore/blob/main/src/Security/README.md
- https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md

We could either add another readme to the samples folder, or at least update the base Security readme to include running the samples.

That said, the instructions in https://docs.microsoft.com/en-us/aspnet/core/security/authentication/samples are misleading, you can't just dotnet run projects in our repo. For prior versions it would be easier for people to copy the samples out of the repo and remove the `Reference`'s.
https://github.com/dotnet/aspnetcore/blob/e897454413fe2103a5b3fdbaacd40472e1cabfd7/src/Security/samples/Cookies/Cookies.csproj#L8-L13
